docs/collect_readmes.py

```diff
@@ -5,6 +5,7 @@
 full_name_lut = {
     'a35t': 'Arty 35T',
     'a100t': 'Arty 100T',
+    'nexys4DDR': 'Nexys 4 DDR',
```

Comment:
This should be `nexys4ddr` as expressed in the README, otherwise I believe a key-not-found error will be generated.

Reply:
Good catch! I fixed the typo.